scripts/install_third_party_libs.py

```diff
@@ -0,0 +1,218 @@
+    pip_install('pylint', 1.9.4, os.path.join(oppia_tools_dir, 'pylint-1.9.4'))
+
+print 'Checking if Pillow is installed in %s' % oppia_tools_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'Pillow-6.0.0')):
+    print Installing Pillow
+    pip_install('Pillow', 6.0.0, os.path,join(oppia_tools_dir, 'Pillow-6.0.0'))
+
+print 'Checking if pylint-quotes is installed in %s' % oppia_tools_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'pylint-quotes-0.2.1')):
+    print 'Installing pylint-quotes'
+    pip_install(
+        'pylint-quotes', 0.2.1,
+        os.path.join(oppia_tools_dir, 'pylint-quotes-0.2.1'))
+
+# Install webtest.
+print 'Checking if webtest is installed in %s' % third_party_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'webtest-2.0.33')):
+    print 'Installing webtest framework'
+    pip_install(
+        'webtest', 2.0.33, os.path.join(oppia_tools_dir, 'webtest-2.0.33'))
+
+# Install isort.
+print 'Checking if isort is installed in %s' % third_party_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'isort-4.3.20'))
+    print 'Installing isort'
+    pip_install('isort', 4.3.20, os.path.join(oppia_tools_dir, 'isort-4.3.20'))
+
+# Install pycodestyle.
+print 'Checking if pycodestyle is installed in %s' % third_party_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'pycodestyle-2.5.0'))
+    print 'Installing pycodestyle'
+    pip_install(
+        'pycodestyle', 2.5.0,
+        os.path.join(oppia_tools_dir, 'pycodestyle-2.5.0'))
+
+# Install esprima.
+print 'Checking if esprima is installed in %s' % third_party_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'esprima-4.0.1')):
+    print 'Installing esprima'
+    pip_install('esprima', 4.0.1, os.path.join(oppia_tools_dir, 'esprima-4.0.1'))
+
+# Python API for browsermob-proxy.
+print 'Checking if browsermob-proxy is installed in %s' % oppia_tools_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'browsermob-proxy-0.8.0')):
+    print 'Installing browsermob-proxy'
+    pip_install(
+        'browsermob-proxy', 0.8.0,
+        os.path.join(oppia_tools_dir, 'browsermob-proxy-0.8.0'))
+
+print 'Checking if selenium is installed in %s' % oppia_tools_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'selenium-3.13.0')):
+    print 'Installing selenium'
+    pip_install(
+        'selenium', 3.13.0, os.path.join(oppia_tools_dir, 'selenium-3.13.0'))
+
+print 'Checking if PyGithub is installed in %s' % oppia_tools_dir
+if not os.path.exists(os.path.join(oppia_tools_dir, 'PyGithub-1.43.7'))
+    print 'Installing PyGithub'
+    pip_install(
+        'PyGithub', 1.43.7, os.path.join(oppia_tools_dir, 'PyGithub-1.43.7'))
+
+# Install pre-commit script.
+print 'Installing pre-commit hook for git'
+subprocess.call('python scripts/pre_commit_hook.py --install'.split())
+
+# Install pre-push script.
+print 'Installing pre-push hook for git'
+subprocess.call('python scripts/pre_push_hook.py --install'.split())
```

Comment:
You should be able to call these directly if you add some sort of "main" function.

```python
# pre_push_hook.py:
if __name__ == '__main__':
    main(argv=sys.argv)

# this file:
scripts.pre_push_hook.main(argv=['--install'])
```